### ISSUE TITLE
Secret Saturday menu breaks bot

### ISSUE THREAD

**Issue by rayhaanj** [jinankjain/mensaBot#1]
So the secret Saturday menu isn't on the website.
```
➜  mensaBot git:(master) ✗ python bot.py
samstag
Traceback (most recent call last):
  File "bot.py", line 46, in <module>
    menu_div.pop(0)
IndexError: pop from empty list
```

**Comment by jinankjain** [comment]
Yeah that needs a fix

**Comment by jinankjain** [comment]
Fixed by #2 . Thanks to @rayhaanj  :+1:  Closing this now.
